src/api/useMeasureServiceApi.tsx

```diff
@@ -497,6 +497,26 @@ export class MeasureServiceApi {
     }
   }
 
+  async unshareMeasures(measureUserIdMap: Map<string, string[]>): Promise<any> {
+    try {
+      const response = await axios.put(
+        `${this.baseUrl}/measures/unshared`,
+        Object.fromEntries(measureUserIdMap),
+        {
+          headers: {
+            Authorization: `Bearer ${this.getAccessToken()}`,
+          },
+        }
+      );
+      return response.data;
+    } catch (err) {
+      const message =
+        "Unable to unshare the selected measure(s) with the users who were unchecked. If the error persists, please contact the help desk.";
+      console.error(message, err);
+      throw new Error(message);
```

Comment:
We already do this approach in other places as well, but personally I am not a big fan. By generating a new custom message we are ignoring the Error messages our API is sending, For example if a User is not authorized to perform this action, then the "err" object will have that information, in that case we should be displaying the error Message our API generated.